ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/ReadWriteServiceBridge.scala

```diff
@@ -115,17 +115,16 @@ case class ReadWriteServiceBridge(
     //   no beginAfter supported
     //   neither multiple subscriptions
     //   neither bootstrapping the bridge from indexer persistence
```

Comment:
```suggestion
    //   This method may only be called once, either with `beginAfter` set or unset.
    //   A second call will result in an error unless the server is restarted.
    //   Bootstrapping the bridge from indexer persistence is not supported either.
```

Reply:
@nmarton-da, how's this?

Reply:
first part works, the last part is actually works now, so I would do `Bootstrapping the bridge from indexer persistence is supported.`